2016-10/october-05.md

```diff
@@ -0,0 +1,55 @@
+## October 05 ([discuss](https://github.com/webpack/meeting-notes/pull/15))
+
+### Attendees
+
+* [Sean Larkin](http://github.com/thelarkinn)
+* [Juho Vepsäläinen](http://github.com/bebraw)
+* [Johannes Ewald](http://github.com/jhnns)
+* [Kees Kluskens](https://github.com/SpaceK33z)
+
+### Agenda
+
+#### Consideration of New Core Team Member
+
+  * Sean: Kees's work on [webpack-dev-server](http://github.com/webpack/webpack-dev-server) and [webpack-dev-middleware](http://github.com/webpack/webpack-dev-middleware) has been impressive and he has continued to offer to help even as these two large repo's themself have been well maintained. This alone is justifyable. 
+
+  * Everyone was in agreeance that Kees is suited. Officially adding him as a member to the webpack core team. 
+
+  * In addition, will give him opportunity to maintain other repo's in the organization if willing. Wanting his focus to still be on webpack-dev-server and webpack-dev-middleware, and only taking on what he can handle, as we have been more than happy with the work he has done. 
```

Comment:
Some time before this meeting we discussed file-loader and url-loader as good candidates that need some attention.